conf/cluster.yaml.sample

```diff
@@ -11,9 +11,9 @@ cluster:
       - name: hare
       - name: s3
         services:
-        - io
-        - auth
-        - bg_consumer
+        - s3server, haproxy
+        - authserver
+        - s3bgworker
```

Comment:
Why using the service name with prefix "s3" when it is already marked under component "s3"
Consider using name such as "bg_worker" or "bgworker"

Reply:
@nileshgovande  , @vimalraghwani20  ^^

Reply:
lets call it bgworker